app/models/aggregated_cash_outgoings.rb

```diff
@@ -0,0 +1,36 @@
+class AggregatedCashOutgoings < BaseAggregatedCashTransaction
+  def self.cash_transaction_categories
+    %i[
+      rent_or_mortgage
+      child_care
+      maintenance_out
+      legal_aid
+    ].freeze
```

Comment:
Should there be a test to check that these cash_transaction_categories exist in TransactionType::NAMES[:debit] ?

If so, could you also do this for AggregatedCashIncome please and thank you